### DISCUSSION TITLE
Help me unsubscribe

### DISCUSSION THREAD

**Discussion by sergiotechrebel** [vercel/community#6697] [Account Issue] [answered]
### Question

I am unable to unsubscribe. In the scope selector I see that I am in the Hobby plan, but I am getting  $20 invoices. For example  invoice #AA3C2BCD-0007 . Please help me cancel my subscription.
--

<br class="Apple-interchange-newline">
--

<br class="Apple-interchange-newline">

**Comment by amyegan** [comment, marked as answer]
Hey @sergiotechrebel. Please [submit an **Invoice Enquiry** case](https://vercel.com/help) to the Customer Success Team. That will get you in touch with the specialists who can check billing information to help with this 🙇
